### DISCUSSION TITLE
Non-RGB light bulb?

### DISCUSSION THREAD

**Discussion by stijnb1234** [make-all/tuya-local#721] [General]
I have some generic A60 CCT lights, but I can only choose a RGBCW type. Is it possible to change the CCT without the RGB?

**Comment by make-all** [comment]
Currently there are only configs for dimmable single white colour and RGBCW bulbs.  Some of the RGBCW bulbs only report the dps for the mode they are currently in, so due to optional dps they will match with CCT lights with similar dps layout even if the light is only a subset of the config's features.

A new config for such lights would be best.  Because they will only support white mode, the color_mode dp should probably be renamed to work_mode to avoid misusing it (similar to some of  the dimmable bulbs), and just the switch, brightness and color_temp dps set (maybe timer and any config options it has).

**Reply by stijnb1234** [reply]
I agree, currently I can change the RGB color on a light that doesn't support RGB 😛
